### PR TITLE
Fixing yarn check in create-falcon-app

### LINKED ISSUE #667: npx with yarn creates empty root node_modules
**What should be changed/improved?**
grafik

Currently the npx command using yarn creates a root node_modules directory because normally there is a root directory and a package.json but in the example the subdirectories server and client are actually used.

### PR DESCRIPTION
**Pull request by Borales**
### Please check if the PR fulfills these requirements ###

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Changelog have been added / updated (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information:**
Fix #667

### CODE REVIEW

**Comment by nimobeeren** [comment]
Changes look good, but how can I test the new version of create-falcon-app? I'd guess `npx` just takes the latest version from npm.

File: yarn.lock [hunk 1]
```diff
@@ -6901,6 +6889,15 @@ cross-spawn@^5.0.1, cross-spawn@^5.1.0:
     shebang-command "^1.2.0"
     which "^1.2.9"
 
+cross-spawn@^7.0.0:
```
Comment (tutok) [inline yarn.lock]: hmm, we already using `cross-spawn-promise` (in `falcon-client`) maybe we could choose one, `cross-spawn-promise` or `exaca`, we will have less dependencies?

File: packages/create-falcon-app/package.json [hunk 2]
```diff
@@ -15,12 +15,12 @@
...
   "dependencies": {
-    "chalk": "2.4.1",
+    "chalk": "2.4.2",
     "commander": "2.19.0",
-    "execa": "1.0.0",
-    "fs-extra": "7.0.0",
-    "listr": "0.14.2",
-    "node-machine-id": "^1.1.10"
+    "execa": "3.2.0",
+    "fs-extra": "8.1.0",
```
Comment (tutok) [inline packages/create-falcon-app/package.json]: I think we should update also 
image

Comment (Borales) [reply packages/create-falcon-app/package.json]: fixed (bumped to 8.1.0 in all packages)

**Comment by Borales** [comment]
@nimobeeren I just pushed my branch to the "nightly" tag, so get the issue - check `npx create-falcon-app test-bug` and `npx create-falcon-app@nightly test-correct` to check the fixed solution. @tutok please do it also, to check new behavior on Windows.

File: yarn.lock (hunk 1 above)
Comment (Borales) [reply yarn.lock]: removed both

**Comment by tutok** [comment]
ok :+1

**Pull request merged**